GraknConsole.java

```diff
@@ -171,6 +165,7 @@ private boolean runTransactionRepl(GraknClient client, String database, GraknCli
                     break;
                 }
                 if (command.isExit()) {
+                    shutdownQueryJobs();
```

Comment:
Here do we want to shut down the console or just quit the transaction REPL? If the latter, we shouldn't shut down `executorService`, since it is meant to be long-lived along with Console. Right @VladGan ?

Reply:
@lolski We want to shutdown the console. If we want to quite transaction REPL, we should use `close`.

Reply:
@lolski yes, @lriuui0x0 make sure it doesn't shutdown `executorService ` if you go from transaction to `>` level